README.md

```diff
@@ -92,36 +92,84 @@ The blockchain verification algorithm is used
    ```
 3. Token verification
 
+   Init Verifier configuration with embeded keys:
+   
+    ```go
+    resolvers := map[string]pubsignals.StateResolver{
+      "privado:main": state.ETHResolver{
+         RPCUrl:          "https://rpc-mainnet.privado.id",
+         ContractAddress: common.HexToAddress("0x3C9acB2205Aa72A05F6D77d708b5Cf85FCa3a896"),
+      },
+    }
+    verifier, err := auth.NewVerifier(loaders.NewEmbeddedKeyLoader(), resolvers)
+   
+   ```
    Init Verifier:
    
    ```go
-   var verificationKeyloader = &loaders.FSKeyLoader{Dir: keyDIR}
-   resolver := state.ETHResolver{
-      RPCUrl:          <polygon_node_http>,
-      ContractAddress: <state_contract_address>,
-   }
-
-   resolvers := map[string]pubsignals.StateResolver{
-      "polygon:mumbai": resolver,
-   }
-   verifier,err := auth.NewVerifierWithExplicitError(verificationKeyloader, loaders.DefaultSchemaLoader{IpfsURL: "<IPFS NODE HERE>"}, resolvers)
-   // or use NewVerifier and check that verifier instance is not nil. IPFS merklization is not worked without setuping global loader
-   // verifier := auth.NewVerifier(verificationKeyloader, loaders.DefaultSchemaLoader{IpfsURL: "ipfs.io"}, resolvers)
+     var verificationKeyloader = &loaders.FSKeyLoader{Dir: keyDIR}
+     resolver := state.ETHResolver{
+        RPCUrl:          <polygon_node_http>,
+        ContractAddress: <state_contract_address>,
+     }
+
+     resolvers := map[string]pubsignals.StateResolver{
+       "polygon:mumbai": resolver,
```

Comment:
`polygon:mumbai` is not supported, could you update to `polygon:amoy`

Reply:
fixed